samples/workload/XNNPACK/build.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+
+#######################################
+#   build xnnpack benchmark samples   #
+#######################################
+BUILD_SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+WAMR_DIR="${BUILD_SCRIPT_DIR}/../../.."
+WAMR_PLATFORM_DIR="${WAMR_DIR}/product-mini/platforms"
+WAMRC_DIR="${WAMR_DIR}/wamr-compiler"
+OUT_DIR="${BUILD_SCRIPT_DIR}/out"
+
+set -xe
+
+# 1. build xnnpack wasm files
+# 1.1 to decide whether needs to build xnnpack wasm files
+cd ${WAMR_DIR}/samples/workload/XNNPACK
+xnnpack_out_dir=xnnpack/bazel-bin
+if [ -d "$xnnpack_out_dir" ]; then
+  echo "XNNPACK build directory exists"
+  file_count=$(ls "$xnnpack_out_dir" | grep ".wasm$" | wc -l)
+  # TODO: currently should be 116 files, may changes in the future
+  if [ $file_count -eq 116 ]; then
+    echo "Have fully built XNNPACK benchmark wasm files before, rebuild it to update"
+    need_fresh_build=false
+    cmake -B build
+    cmake --build build
+  else
+    echo "Partially build XNNPACK wasm files, start a fresh build"
+    need_fresh_build=true
+  fi
+else
+  echo "XNNPACK build directory does not exist, start a fresh build"
+  need_fresh_build=true
+fi
+
+# 1.2 only build wasm files if needed
+if [ "$need_fresh_build" = true ]; then
+  echo "Start building xnnpack"
+  rm -fr build && mkdir build
+  cmake -B build
+  cmake --build build
+fi
+
+# 1.3 copy files to out/
+rm -rf ${OUT_DIR}
+mkdir ${OUT_DIR}
+cp xnnpack/bazel-bin/*.wasm out/
+
+# 2. compile xnnpack benchmark wasm files to benchmark aot files with wamrc
+# 2.1 build wamr-compiler if needed
+cd ${WAMRC_DIR}
```

Comment:
maybe some `test` first to achieve "build if needed"

Reply:
And if *.wasm -> *.aot is necessary, we shall add it in *CMakeLists.txt*

Reply:
SIMD feature is only supported on aot mode, so maybe it's a good idea to add steps `building wamrc` and `translate wasm to aot` to CMakeLists.txt?